### PR TITLE
attempt fix ling instant revive

### LINKED ISSUE #42360: Lings get an instant revive
Repro steps:
- Become a ling
- Die
- Use reviving stasis
- When the stasis finishes, it BOTH changes your stasis button to the revive button AND revives you, letting you come back to life while keeping the revive button. *It should not be auto reviving you when the stasis finishes*
- You can now press the revive button at any time to instantly revive again

### PR DESCRIPTION
**Pull request by ghost**
:cl: 0d0be32
fix: lings shouldn't instant revive any more
/:cl:

Fixes https://github.com/tgstation/tgstation/issues/42360

probably should test merge and see if this actually fixes it

### CODE REVIEW

**Comment by OverlyGeneric** [comment]
Is this a rebalance PR? I don't like when people feign balancing PRs as fixes.

**Comment by 0d0be32b607ee4ecbc2358f3dd4a27de** [comment]
@OverlyGeneric https://github.com/tgstation/tgstation/issues/42360

**Comment by 0d0be32b607ee4ecbc2358f3dd4a27de** [comment]
considering it took so long to be reported, i would not be surprised if it was a "rare but it happens" thing

**Comment by 0d0be32b607ee4ecbc2358f3dd4a27de** [comment]
tested it workss

File: code/modules/antagonists/changeling/powers/fakedeath.dm [hunk 1]
```diff
@@ -26,6 +33,9 @@
 	return TRUE
 
 /datum/action/changeling/fakedeath/proc/revive(mob/living/user)
+	if(!user || !istype(user))
+		message_admins("Warning! Ling revive user is invalid! Check runtime logs!")
+		log_game("Warning! Ling revive user is invalid! Check runtime logs!")
```
Comment (Jordie0608) [inline code/modules/antagonists/changeling/powers/fakedeath.dm]: Replace these with a return, no need to log or message admins and it doesn't actually stop a runtime.

**Review by Jordie0608 — CHANGES_REQUESTED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
granny pr

**Pull request closed without merging**